lib/dotcom_web/components/trip_planner/trip_planner_results_section.ex

```diff
@@ -39,19 +41,41 @@ defmodule DotcomWeb.Components.TripPlanner.TripPlannerResultsSection do
           </button>
         </div>
       </.async_result>
+      <.async_result :let={results} assign={@results}>
+        <%= if results && @itinerary_selection != :summary do %>
+          <% {:detail, %{itinerary_group_index: itinerary_group_index, itinerary_index: itinerary_index}} = @itinerary_selection %>
+          <% itinerary = Enum.at(results, itinerary_group_index).itineraries |> Enum.at(itinerary_index) %>
+          <% itinerary_map = TripPlan.Map.itinerary_map(itinerary) %>
+          <% lines = TripPlan.Map.get_lines(itinerary_map) %>
+          <% points = TripPlan.Map.get_points(itinerary_map) %>
+          <.live_component
+            module={MbtaMetro.Live.Map}
+            id="trip-planner-map"
+            class={[
+              "h-64 md:h-96 w-full",
+              "relative overflow-none row-span-2",
+              @itinerary_selection == :summary && "hidden md:block"
+            ]}
+            config={@map_config}
+            lines={lines}
+            pins={[@from, @to]}
+            points={points}
+          />
+        <% else %>
+          <.live_component
+            module={MbtaMetro.Live.Map}
+            id="trip-planner-map"
+            class={[
+              "h-64 md:h-96 w-full",
+              "relative overflow-none row-span-2",
+              @itinerary_selection == :summary && "hidden md:block"
+            ]}
+            config={@map_config}
+            pins={[@from, @to]}
+          />
+        <% end %>
```

Comment:
📓 None of this will be here after we change the state management. It must be embedded like this because the results component includes the map.